diagnostic.md

```diff
@@ -32,7 +32,21 @@ format described in the instructions). The page should have a title in the head
 and an unordered list (with three items) inside the body.
 
 <!-- your answer starts here -->
-
+```HTML
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Restaurant Website</title>
+  </head>
+  <body>
+    <ul>
+      <li>Breakfast Menu
+      <li>Lunch Menu
+      <li>Dinner Menu
+    </ul>
+  </body>
+</html>
+```
```

Comment:
We learned in class that the closing tags are optional. Documentation on MDN says: "The end tag can be omitted if the list item is immediately followed by another <li> element, or if there is no more content in its parent element." Since these list items seem to fit this criteria, are the closing tags still necessary?